site/en/docs/extensions/reference/commands/index.md

```diff
@@ -134,10 +134,8 @@ chrome.commands.onCommand.addListener((command) => {
 
 ### Action commands
 
-The `_execute_action` (Manifest V3), `_execute_browser_action` (Manifest V2), and
```

Comment:
Given this is the API reference (which doesn't have an MV2 variant), I think we should actually keep both for now?